crates/levm/src/call_frame.rs

```diff
@@ -1,6 +1,14 @@
 use crate::{memory::Memory, opcodes::Opcode};
 use bytes::Bytes;
-use ethereum_types::{Address, U256};
+use ethereum_types::{Address, H32, U256};
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Hash)]
+/// Data record produced during the execution of a transaction.
+pub struct Log {
+    pub address: Address,
+    pub topics: Vec<H32>,
+    pub data: Vec<u8>,
```

Comment:
Let's use `Bytes` to keep consistency.

```suggestion
    pub data: Bytes,
```